persist_error/handler.py

```diff
@@ -51,8 +52,10 @@ def lambda_handler(event, context):
 
         try:
             json_file = initial_input['Record']['s3']['object']['key']
+            s3_url = f'https://s3.console.aws.amazon.com/s3/object/iow-retriever-capture-{deploy_stage.lower()}?region={region}&prefix={json_file}'
```

Comment:
You can actually pull the bucket name out of the initial input rather than constructing via `initial_input['Record']['s3']['bucket']['name']`.